python/lsst/ctrl/events/eventsLib.i

```diff
@@ -61,15 +61,24 @@ Access to the lsst::ctrl::events classes
 
 %shared_ptr(lsst::ctrl::events::EventFormatter)
 
+
 %import "lsst/daf/base/baseLib.i"
 %import "lsst/pex/logging/loggingLib.i"
-%import "lsst/pex/policy/policyLib.i"
 
+%typemap(out) std::vector<std::string > {
```

Comment:
I used this because it was the method used in another package (pex_policy, I believe) which was "automagically" preforming this operation for me on it's inclusion.  When I got rid of pex_policy references I went to see how to do this, since I started getting a tuple back instead of a list..  If this line can replace it, I'll use that instead.